pkg/kubelet/stats/stats_provider.go

```diff
@@ -96,11 +96,8 @@ func (p *StatsProvider) GetCgroupStats(cgroupName string) (*statsapi.ContainerSt
 	if err != nil {
 		return nil, nil, fmt.Errorf("failed to get rootFs info: %v", err)
 	}
-	imageFsInfo, err := p.cadvisor.ImagesFsInfo()
-	if err != nil {
-		return nil, nil, fmt.Errorf("failed to get imageFs info: %v", err)
-	}
-	s := cadvisorInfoToContainerStats(cgroupName, info, &rootFsInfo, &imageFsInfo)
+	// Imagefs doesn't make sense for raw container.
+	s := cadvisorInfoToContainerStats(cgroupName, info, &rootFsInfo, nil)
```

Comment:
Given that we will return partial ContainerStats, does it also make sense to leave rootFs empty?

Reply:
I don't quite get it. :)

Reply:
Sorry, I didn't make it clear. Looks like the only callers of `GetCgroupStats` are in https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/server/stats/summary.go. None of them use the filesystem stats. Should we just not get those stats and document the missing fs stats for the function?

Reply:
Done.